app/shared/language/language-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { LanguageSchema } from 'shared/types/commonTypes';
+import { LanguageCode, availableLanguages, OTHER_LANGUAGE_SCHEMA } from './available-languages';
+
+class LanguageUtils {
+  static getLanguagesByCode(code: LanguageCode = 'elastic'): LanguageSchema[] {
```

Comment:
If these are already LanguageUtils, I'm wondering if the methods should include the term `Language` or `Languages` still?  For example `LanugageUtils.getByCode` may be all that is needed.  Same as `LanguageUtils.getCodes` or `LanguageUtils.getSchema`.  Not sure about this though.  Mabye it's OK as is.

Reply:
Good point, since this is under the language context, clients consuming this methods will have enough idea just by seeing `LanguageUtils.getCodes`

Going to refactor it.